ext/exporters.d.ts

```diff
@@ -1,6 +1,7 @@
 declare const exporters: {
   LOG: 'log',
-  AGENT: 'agent'
+  AGENT: 'agent',
+  CI: 'ci'
```

Comment:
Could this have a name more in line with what it does and not specific to a product? For example "agentless" or "intake". Maybe even "datadog" since the data is sent to Datadog directly. We'll probably want to reuse that exporter when we go fully agentless.